api/api.go

```diff
@@ -0,0 +1,288 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the GPLv3, see LICENCE file for details.
+
+// The api package contains the interface and implementation of the
+// terms service client.
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/juju/errors"
+	rterms "github.com/juju/romulus/api/terms"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+
+	"github.com/juju/terms-client/api/wireformat"
+)
+
+const baseURL = "https://api.jujucharms.com/terms"
+
+// Client reprensents the interface of the terms service.
```

Comment:
represents (spelling)
Also, probably better to say something like "Client represents the terms service API."